crate2nix/templates/build.nix.tera

```diff
@@ -15,6 +15,7 @@
 , rootFeatures ? [ "default" ]
   # If true, throw errors instead of issueing deprecation warnings.
 , strictDeprecation ? false
+, getCleanedSource ? null
```

Comment:
Please add a comment here explaining the usage of the attribute.

Reply:
Something in the README.md would also be appreciated!